lib/associations/belongs-to.js

```diff
@@ -78,19 +79,21 @@ class BelongsTo extends Association {
        * @param {Object} [options]
        * @param {String|Boolean} [options.scope] Apply a scope on the related model, or remove its default scope by passing false.
        * @param {String} [options.schema] Apply a schema on the related model
-       * @see {Model#findOne} for a full explanation of options
-       * @return {Promise<Instance>}
+       * @see {@link Model.findOne} for a full explanation of options
```

Comment:
Probably not - I think jsdoc has some convention about one being instance and the other being static members, but I'm not sure